src/DependencyInjection/EightPointsGuzzleExtension.php

```diff
@@ -82,7 +82,7 @@ public function load(array $configs, ContainerBuilder $container)
             // if present, add default options to the constructor argument for the Guzzle client
             if (isset($options['options']) && is_array($options['options'])) {
                 foreach ($options['options'] as $key => $value) {
-                    if ($value === null || (is_array($value) && count($value) === 0)) {
+                    if ($key === 'handler' || $value === null || (is_array($value) && count($value) === 0)) {
```

Comment:
if handler will be on level higher, this condition become useless

Reply:
Done.